airbyte-integrations/connectors/source-airtable/source_airtable/manifest.yaml

```diff
@@ -0,0 +1,142 @@
+          http_method: GET
+        partition_router:
+          type: SubstreamPartitionRouter
+          parent_stream_configs:
+            - type: ParentStreamConfig
+              parent_key: id
+              partition_field: id
+              stream:
+                $ref: "#/definitions/streams/tables"
+        record_selector:
+          extractor:
+            type: DpathExtractor
+            field_path: ["records"]
+      schema_loader:
+        type: DynamicSchemaLoader
+        stream:
+          type: DeclarativeStream
+          name: tables
+          $parameters:
+            table_id: "table_id"
+          retriever:
+            requester:
+              $ref: "#/definitions/base_requester"
+              path: meta/bases/{{stream_partition['id']}}/tables
+              http_method: GET
+            partition_router:
+              type: SubstreamPartitionRouter
+              parent_stream_configs:
+                - type: ParentStreamConfig
+                  parent_key: id
+                  partition_field: id
+                  stream:
+                    $ref: "#/definitions/streams/bases"
+            record_selector:
+              extractor:
+                type: DpathExtractor
+                field_path: ["tables"]
+              record_filter:
+                condition: "{{ record['id'] == parameters['table_id'] }}"
+        schema_pointer: ["fields"]
+        key_pointer: ["name"]
+        type_pointer: ["type"]
+        types_map:
+          - target_type: string
+            current_type: singleLineText
+          - target_type: string
+            current_type: multilineText
+  base_requester:
+    type: HttpRequester
+    url_base: https://api.airtable.com/v0
+    authenticator: "#/definitions/authenticator"
+  # Authenticators
+  bearer_authenticator:
+    type: BearerAuthenticator
+    api_token: "{{ config['credentials']['api_key'] }}"
+  oauth_authenticator:
+    type: OAuthAuthenticator
+    refresh_request_body: {}
+    token_refresh_endpoint: https://airtable.com/oauth2/v1/token
+    grant_type: refresh_token
+    client_id: '{{ config["credentials"]["client_id"] }}'
+    client_secret: '{{ config["credentials"]["client_secret"] }}'
+    refresh_token: '{{ config["credentials"]["refresh_token"] }}'
+  authenticator:
+    type: SelectiveAuthenticator
+    authenticator_selection_path: ["credentials", "auth_method"]
+    authenticators:
+      oauth2: "#/definitions/oauth_authenticator"
+      api_key: "#/definitions/bearer_authenticator"
+
+streams:
+  - $ref: "#/definitions/streams/custom_object"
+
+dynamic_streams:
+  - type: DynamicDeclarativeStream
+    stream_template:
+      $ref: "#/definitions/streams/custom_object"
+    components_parser:
+      type: DynamicComponentsParser
+      components_values_stream:
+        $ref: "#/definitions/streams/tables"
+      components_mapping:
+        - type: MapComponentsDefinition
+          key: name
+          value: "{{ components_value['name'] }}"
```

Comment:
Brainstrorming: To avoid interpolation here, we could use the same pattern as `RequestOption.inject_into`

Reply:
`inject_into` specifies the types of destinations where data will be pasted but does not define the sources of data. Could you provide more details or examples to clarify how the sources of data are determined and how they relate to the destinations?

Reply:
I think we will have different than `components_value` when parsing the config for custom reports, which could vary, it’s better to make this an interpolated string. This way, it can be reused in other classes that don’t make calls but iterate through the config-defined items.